api/indexer.oas2.json

```diff
@@ -2677,6 +2738,61 @@
     }
   },
   "parameters": {
+    "proposer": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "x-algorand-format": "Address"
+      },
+      "description": "Account(s) marked as proposer in the block header's participation updates. This parameter accepts a comma separated list of addresses.",
```

Comment:
I think it's fine, maybe even more correct, to simply say "Accounts" here, where the number is unknown.

Reply:
Addressed in https://github.com/algorand/indexer/pull/1638/commits/a302e5b9948fa1dace8f77f2b31bfeb8227d0bac